Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -9,13 +9,13 @@ file that was distributed with this source code.
 
 #}
 {% macro render_array(value, inline) %}
+    {% from _self import render_array %}
     {% for key, val in value %}
         {% if val is iterable %}
-            [{{ key }} => {{ _self.render_array(val, inline) }}]
+            [{{ key }} => {{ render_array(val, inline) }}]
         {%  else %}
             [{{ key }} => {{ val }}]
         {%  endif %}
-
```

Comment:
Why did you remove this line?

Reply:
this was added to support both array and string in single macro, my concept was in case there is mixed data ( array / string ) then macro could handle both easily, but as you recommended that it should be one commit per functional change so I did it, sorry If i did something wrong, I am not expert in using github

Reply:
Can you add the line back then?